packages/lib/slots.ts

```diff
@@ -52,14 +56,14 @@ const getSlots = ({ inviteeDate, frequency, minimumBookingNotice, workingHours,
     return [];
   }
 
-  const localWorkingHours = getWorkingHours(
-    { utcOffset: -inviteeDate.utcOffset() },
-    workingHours.map((schedule) => ({
-      days: schedule.days,
-      startTime: startOfDay.add(schedule.startTime, "minute"),
-      endTime: startOfDay.add(schedule.endTime, "minute"),
-    }))
-  ).filter((hours) => hours.days.includes(inviteeDate.day()));
+  const workingHoursUTC = workingHours.map((schedule) => ({
+    days: schedule.days,
+    startTime: /* Why? */ startOfDayUTC.add(schedule.startTime, "minute"),
```

Comment:
The getWorkingHours function works with ConfigType instances that can't make sense of startTime given in minutes since the start of the day (in UTC)

Reply:
Would it make sense to assume inside `getWorkingHours` that if the `startTime/endTime` is type of number then we skip all conversions altogether?